src/sys/unix/ready.rs

```diff
@@ -384,8 +378,8 @@ impl fmt::Debug for UnixReady {
             (UnixReady(Ready::writable()), "Writable"),
             (UnixReady::error(), "Error"),
             (UnixReady::hup(), "Hup"),
-            #[allow(deprecated)]
-            (UnixReady::aio(), "Aio")];
+            (UnixReady::aio(), "Aio"),
```

Comment:
aio and lio probably shouldn't be included if the platform doesn't support it.

Reply:
Assuming the Debug impl should print Aio and Lio on supported platforms, then the only question is how to implement it. This should work, since on unsupported platforms, `aio()` and `lio()` return `empty()`, which prints nothing. Do you prefer an alternative implementation?